tensorflow/compiler/tf2tensorrt/convert/ops/fill_ops.cc

```diff
@@ -25,10 +25,62 @@ namespace convert {
 
 #if IS_TRT_VERSION_GE(8, 2, 0, 0)
 
-class ConvertFill : public OpConverterBase<ConvertFill> {
+template <typename Impl>
+class ConvertFillBase : public OpConverterBase<Impl> {
+ public:
+  explicit ConvertFillBase(OpConverterParams* params)
+    : OpConverterBase<Impl>(params) {}
+
+  Status AddFill(const TRT_TensorOrWeights& value_input,
```

Comment:
It would make sense to move this function into layer utils as a part of `TRTNetworkBuilder`